lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -131,7 +131,7 @@ class FirstMeaningfulPaint extends Audit {
     score = Math.min(100, score);
     score = Math.max(0, score);
 
-    timings.navStart = data.navStart.ts / 1000;
+    timings.navStart = data.navStart && data.navStart.ts / 1000;
```

Comment:
If `data.navStart` is undefined here, should `timings.navStart` have a default value instead of undefined?